scalafix-rules/src/main/scala/scalafix/internal/rule/DisableSyntax.scala

```diff
@@ -196,6 +196,12 @@ final class DisableSyntax(config: DisableSyntaxConfig)
               "Default args makes it hard to use methods as functions.",
               m.pos)
           }
+      case Term.ApplyInfix(_, t @ Term.Name("=="), _, _) if config.noUniversalEquality =>
```

Comment:
Let's add a case for `Term.Apply` as well since `1.==(2)` is valid

Reply:
Yeah, I've initially excluded it on purpose.

> I've explicitly restricted the rule to infix applications of == as I don't think handling non-infix applications (e.g., 1.==(2)) is worth the (albeit low) effort.

But today is a new day and I've changed my mind.